dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgModels.ts

```diff
@@ -0,0 +1,366 @@
+export const lksgKpis = {
```

Comment:
I added an excel with findings to the JIRA issue

Reply:
names updated according to excel

Reply:
In the last category "Waste" I can see 2 Values having empty name and tool tip. Between "Persistent Organic Pollutants Production And Use Risk For Importing State" and "Hazardous Waste Disposal". Also I noted that the field "Total Revenue" according to the tooltip is to be displayed in millions (MM).

Reply:
fixed